### PR TITLE
ipatests: Test IPACertNSSTrust check when trust attributes is modified for specific cert

### PR DESCRIPTION
**Pull request by menonsudhir**
This test modifies the trust attribute of Server-Cert and checks that healthcheck tool reports correct status

### CODE REVIEW

File: ipatests/test_integration/test_ipahealthcheck.py [hunk 1]
```diff
@@ -760,6 +760,63 @@ def test_chainexpiration_check_without_cert(self, move_ipa_ca_crt):
...
+                "certutil",
+                "-M",
+                "-d",
+                paths.PKI_TOMCAT_ALIAS_DIR,
+                "-n",
+                "Server-Cert cert-pki-ca",
+                "-t",
+                "CTu,u,u",
+                "-f",
+                paths.PKI_TOMCAT_ALIAS_PWDFILE_TXT,
```
Comment (rcritten) [inline ipatests/test_integration/test_ipahealthcheck.py:780]: Black I assume? It is honestly more readable for me to keep option/value pairs together, e.g.

```
"-M",
"-d", paths.PKI_TOMCAT_ALIAS_DIR,
"-n", "Server-Cert cert-pki-ca",
...
```
I also wonder if this fixture would benefit to take the host, db, nickname and trust as options so we could modify multiple values at once. Not a deal breaker, this is the minimum needed to test the check.

Comment (menonsudhir) [reply ipatests/test_integration/test_ipahealthcheck.py]: @rcritten yes i had used black, have made the changes to keep option/value pair together.

**Comment by rcritten** [comment]
This looks fine, please remove the temp commit.

**Comment by flo-renaud** [comment]
Tests are green and PR already reviewed by @rcritten , adding ACK.

**Comment by flo-renaud** [comment]
master:

* bb2dfbbf0b0e3daf7207bf6605b9f23edb115be5 ipatests: Test IPACertNSSTrust check when trust attributes is modified for specific cert

**Pull request closed without merging**